partners/scientific-communities.md

```diff
@@ -52,12 +61,15 @@ Our vibrant community is filled with experts in the Python packaging space. A
 partnership with us means that you will always have support if you run into
 Python packaging issues.
 
-### Future work
+:::{admonition} What's in the works
+:class: note
+
+We are currently enhancing our peer review process and working together on:
```

Comment:
```suggestion
We are currently enhancing our peer review process and collaborating on:
```

Reply:
it was unclear who the "we" is in "working together"

Reply:
totally - i've changed it as follows:

> - Adding an editorial bot to our peer review infrastructure to streamline our review processes.
> - Collecting package metrics to track health and also understand maintenance status.